config.json

```diff
@@ -434,7 +434,11 @@
       "core": true,
       "difficulty": 5,
       "slug": "robot-name",
-      "topics": null,
+      "topics": [
+        "regular_expressions",
+        "strings",
+        "randomness"
```

Comment:
Maybe `text_formatting` and `pattern_matching` could be useful? I think you've included the most important ones though, so I don't mind if you add those two or not :)

Reply:
`text_formatting` is a good idea, I gonna add that later on. 
Isn't `pattern_matching` already covered by regex though?

Reply:
You could try and look for something matching a pattern without using a regex. And some people might be looking for exercises to do with pattern matching without thinking to look for exercises to do with regular expressions. But you're right, they are very similary so I'm happy for you to just stick with `regular_expressions` if you think that's best :)